flake.nix

```diff
@@ -411,12 +411,12 @@
             inherit (packages.proof-proverif) CRYPTOVERIF_LIB;
             inputsFrom = [ packages.default ];
             nativeBuildInputs = with pkgs; [
+              inputs.fenix.packages.${system}.complete.toolchain
```

Comment:
I think it is fine. However, pulling in the full toolchain is fat, i.e. this includes:

- cargo
- cargo-clippy
- cargo-fmt
- cargo-miri
- clippy-driver
- miri
- rls
- rust-analyzer
- rust-gdb
- rust-gdbgui
- rust-lldb
- rustc
- rustdoc
- rustfmt

If we are fine with having a more fat devShell, then the nice thing is that it is really _complete_.

Reply:
@wucke13 Thanks for the review!